veascan-web/src/components/TxCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import styled from "styled-components";
+import Copy from "tsx:../../public/tx-info/Copy.svg";
+import ArbitrumLogo from "tsx:../assets/svgs/tx-info/ArbitrumLogo.svg";
+import EthereumLogo from "tsx:../assets/svgs/tx-info/EthereumLogo.svg";
+
+interface TxCardProps {
+  title: string;
+  chain: string;
+  txHash: string;
+  timestamp: string;
+  caller: string;
+  extraFields?: {
+    [key: string]: string;
+  };
+}
+
+const StyledDiv = styled.div`
```

Comment:
Try using less classnames and using a more "styled-components" approach. For example the .tx-header could be:
```typescript
const Header = styled.label`
  // style
`;
```
I think the result is more readable since when reading the code I know that this style will be applied to a label, with classnames this can be applied to any element.